packages/contentful-extension-scripts/scripts/start.js

```diff
@@ -19,17 +19,19 @@ const options = {
   scopeHoist: false, // Turn on experimental scope hoisting/tree shaking flag, for smaller production bundles
   logLevel: 3, // 5 = save everything to a file, 4 = like 3, but with timestamps and additionally log http requests to dev server, 3 = log info, warnings & errors, 2 = log warnings & errors, 1 = log errors
   hmr: true, // Enable or disable HMR while watching
-  hmrPort: 0, // The port the HMR socket runs on, defaults to a random free port (0 in node.js resolves to a random free port)
+  hmrPort: 54321, // The port the HMR socket runs on, defaults to a random free port (0 in node.js resolves to a random free port)
```

Comment:
I believe that part of the comment is redundant after the change: ```defaults to a random free port (0 in node.js resolves to a random free port)```

Reply:
True! I'll update it.